bio2zarr/writer.py

```diff
@@ -0,0 +1,813 @@
+import dataclasses
```

Comment:
I'm not sure I see the value of this module - why not keep the VCZ writing logic in vcz.py (which we could move to root of the package hierarchy)?